app/views/papers/_sort_by.html.erb

```diff
@@ -0,0 +1,4 @@
+<div class="btn-group btn-group-xs nav-button-group sort" role="group" aria-label="...">
+  <%= link_to 'Votes', page == 'list' ? list_path(sort: 'votes') : papers_path(sort: 'votes'), class: "btn btn-default" + (params[:sort] == "pub_date" ? "" : " active"), role: "button" %>
+  <%= link_to 'Publish Date', page == 'list' ? list_path(sort: 'pub_date') : papers_path(sort: 'pub_date'), class: "btn btn-default" + (params[:sort] == "pub_date" ? " active" : ""), role: "button"  %>
```

Comment:
I was thinking the same thing.  I thought it might be something I handle through routes?  I was sort of working off a railscast tutorial and he handled it through the controller, so I thought I'd start there.  But definitely if there's a better way, we should do that.

I was also thinking this is hard to read.  Should I just stick with multi line if statements?